backend/src/user/user.controller.ts

```diff
@@ -0,0 +1,13 @@
+import {UserService} from "./user.service";
+import {Controller, Post} from "@nestjs/common";
+
+@Controller('createuser')
+export class UserController {
+    constructor(private readonly userService: UserService) {
+    }
+    @Post()
+    createNewUser(email: string, name: string) {
+        const data = {email: email, login: name}
```

Comment:
data to nieładna nazwa, nic nie mówi. Albo te parametry przekazać bezpośrednio i osobno do serwisu, albo niech cała metoda Post w ciele żądania zawiera stosowny obiekt (skłaniałbym się ku tej drugiej opcji)